packages/cloud-cognitive/src/components/CreateTearsheet/preview-components/stepDataTest.js

```diff
@@ -0,0 +1,53 @@
+import { settings } from 'carbon-components';
+import { Tooltip } from 'carbon-components-react';
+
+const { prefix } = settings;
+
+const tooltipOne = () => {
+  return (
+    <Tooltip
+      direction="bottom"
+      showIcon={false}
+      triggerClassName={`${prefix}--progress-label`}
+      triggerText={'Topic name'}
+      tooltipId="tooltipId-1">
+      <p>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Animi
+        consequuntur hic ratione aliquid cupiditate, nesciunt saepe iste
+        blanditiis cumque maxime tenetur veniam est illo deserunt sint quae
+        pariatur. Laboriosam, consequatur.
+      </p>
+    </Tooltip>
+  );
+};
+
+const stepDataTest = [
```

Comment:
I think we would need this step data created with the `CreateTearsheet` component dynamically, otherwise, these will be the steps that render for anyone using this component.

Reply:
The logic that renders that component is now within `CreateInfluencer` and something like this would allow you to get the shape you need

```js
let stepData = [];
createComponents.steps.forEach(step => {
    stepData = [...stepData, {
      label: step.props?.title,
      description: step.props?.title,
      secondaryLabel: step.props?.secondaryLabel
    }]
});
```